pkgs/tools/security/otpgen/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, makeWrapper
+, fetchFromGitHub
+, fetchpatch
+, cmake
+, pkg-config
+, cryptopp
+, qtbase
+}:
+stdenv.mkDerivation rec {
+
```

It was actually this one:

```suggestion

stdenv.mkDerivation rec {
```